velox/substrait/SubstraitToVeloxPlanValidator.cpp

```diff
@@ -43,14 +44,20 @@ bool SubstraitToVeloxPlanValidator::validateInputTypes(
     std::vector<TypePtr>& types) {
   // The input type is wrapped in enhancement.
   if (!extension.has_enhancement()) {
+    validateLog_.emplace_back(
```

Comment:
I think it may be better to use a uniform API for logging, even though it is quite simple, e.g. `logValidateMsg(std::string& msg)`. And it can also make the log behavior more controllable.

Reply:
What I want to do is to consolidate log from java side and native side and the print will be controlled at java side. Not print them separately.

Reply:
What I meant is not about the printing behavior, but the log collecting behavior. With a uniform API, we can disable it easily in the future if user wants. Right?

Reply:
so what will you want to do in this method `logValidateMsg(std::string& msg)`?  print it or collect it them similar as `validateLog_`? Current we do the collect job through adding it to `validateLog_`. What do you mean by `disable` it? not collecting them anymore or disable the print? if it's the later, now we have a conf at java side to control all fallback message print.

Reply:
logValidateMsg still for collecting.